RESPONSIBILITIES.md

```diff
@@ -56,6 +57,17 @@ It's our responsibility to ensure the content and code in pull requests are corr
 - If a pull request is valuable but isn't gaining traction, consider reaching out to fulfill the necessary requirements. This way, the pull request can be merged, even if the work is done by several individuals.
 - Lastly, strive for progress, not perfection.
 
+### Merging a Pull Request 
+
+It is important that commit messages are helpful in understanding the reasons for a given commit and maintain good commit hygiene by only keeping the relevant information. 
+
+Most repositories in [opensearch-project](https://github.com/opensearch-project) are configured to require commits to be squashed into a single commit when merging pull requests. This process needs human intervention to produce high quality commit messages. When merging pull requests, edit merge messages by following these steps as much as possible:
+
+- The commit subject should be concise and clearly convey what is being merged.
+- The commit body should include the details (if any) about the commit, typically inline with the PR description.
```

Comment:
I very much like for commit subjects to limited to about 50 characters and bodies wrapped at around 72 characters, though neither are hard maximums. This helps for the commit messages to work well with all the git tooling. Just my opinion, but maybe worth adding as a suggestion here?

Reply:
I have added a brief note in the guidelines as of now. Let me know if that makes sense.